src/game/OutdoorPvP/OutdoorPvPEP.cpp

```diff
@@ -43,11 +43,14 @@ OutdoorPvPEP::OutdoorPvPEP() : OutdoorPvP(),
 
 void OutdoorPvPEP::FillInitialWorldStates(WorldPacket& data, uint32& count)
```

Comment:
same here, the OutdoorPVPEP class should never get an object because it's disabled through config